code/_onclick/hud/fullscreen.dm

```diff
@@ -156,9 +156,11 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/fullscreen)
 	icon_state = "noise"
 
 /atom/movable/screen/fullscreen/high
-	icon = 'icons/hud/screen_gen.dmi'
-	screen_loc = "WEST,SOUTH to EAST,NORTH"
 	icon_state = "druggy"
+	alpha = 255
+	plane = LIGHTING_PLANE
+	layer = LIGHTING_ABOVE_ALL + 1 //Infinity plus one (not actually)
```

Comment:
why?

Reply:
Any lighting sources above the layer of the druggy hud effect will not be affected by the color changes. This prevents any issues where some lighting sources are not affected by the druggy effect.